ush/retrieve_data.py

```diff
@@ -226,13 +226,14 @@ def fill_template(template_str, cycle_date, templates_only=False, **kwargs):
     return template_str.format(**format_values)
 
 
-def create_target_path(target_path):
+def create_target_path(target_path, cla, mem):
 
     """
     Append target path and create directory for ensemble members
     """
     if not os.path.exists(target_path):
         os.makedirs(target_path)
+    setattr(cla, 'output_path_{0}'.format(mem), target_path)
```

Comment:
The modification here shouldn't be necessary given the mods below, whether the suggested change is adopted or not.

I think it night be best to leave this one with its one job.